tests/test_format_phone_number_list.py

```diff
@@ -0,0 +1,79 @@
+
+import pytest
+
+from phonenumberfmt import format_phone_number_list
+from phonenumberfmt.country import dialing_prefix_for_region
+
+
+@pytest.mark.parametrize("phone_region", ["AE", "PK", "SA", "JO"])
+@pytest.mark.parametrize(
+    "input,output",
+    [
+        # empty phone numbers
+        (None, []),
+        ("-", []),
+        ("--", []),
+        ("No", []),
+        ("", []),
+        # phone number without a country prefix
+        ("55-4396334", ["PREFIX554396334"]),
+        ("+55-6934369", ["PREFIX556934369"]),
+        ("5(5-4(//39(63)3--4", ["PREFIX554396334"]),
+        # some crazy combos
+        ("+40773818041", ["+40773818041"]),
+        ("+31318655336", ["+31318655336"]),
+        ("+97172273000", ["+97172273000"]),
+        ("+92238923892", ["+92238923892"]),
+        ("+40773818041/+97172273000", ["+40773818041", "+97172273000"]),
+        ("+40773818041 / +97172273000", ["+40773818041", "+97172273000"]),
+        ("+971-50-1896203", ["+971501896203"]),
+        ("+971(0)508882395", ["+971508882395"]),
+        (
+            "+971-50-1003023 / +971-558683397",
+            ["+971501003023", "+971558683397"],
+        ),
+        ("971-50-1896203", ["+971501896203"]),
+        ("+92 321 9224895", ["+923219224895", "+9224895"]),
+        (
+            "+971-50-2516838, +971- 55-4396334/ 693436544",
+            ["+971502516838", "+971554396334", "PREFIX693436544"],
+        ),
+        ("////////+40773818041", ["+40773818041"]),
+        ("1234/    +40773818041", ["+40773818041"]),
+        ("+.4.0.7.7.3.8.1.8.0.4.1++++++++++", ["+40773818041"]),
+        ("40773818041", ["+40773818041"]),
+        ("971501230869", ["+971501230869"]),
+        ("00962799506073", ["+962799506073"]),
+        ("00971529010000", ["+971529010000"]),
+        ("%2b97144298820", ["+97144298820"]),
+        (
+            "+97144220125#40,+971503292710#40",
+            [
+                "+97144220125",
+                "+9714422012540",
+                "+971503292710",
+                "+97150329271040",
+            ],
+        ),
+        (
+            "+97144220125:40,+971503292710:40",
+            [
+                "+97144220125",
+                "+9714422012540",
```

Comment:
The output here is wrong. This should be two numbers. `:40` should simply be stripped. That also goes for the previous test case.

Reply:
done